python/bifrost/blocks/guppi_raw.py

```diff
@@ -59,7 +59,7 @@ def on_sequence(self, reader, sourcename):
         df_MHz = bw_MHz / nchan
         f0_MHz = cfreq_MHz - 0.5*(nchan-1)*df_MHz
         # Note: This will be negative if OBSBW is negative, which is correct
-        dt_s   = 1. / df_MHz / 1e6
+        dt_s   = np.abs(1. / df_MHz / 1e6)
```

Comment:
With a reversed channel axis (-ive scale), the fine_time axis is also reversed (-ive scale). The FFT preserves the sign of the axis scale, so if you FFT+fftshift the fine_time axis, then the result will still have -ive scale, and so it can be merged with the channel axis (both have -ive scales).

This means the fine_time axis is reversed relative to the coarse time (block) axis, so merging fine_time and coarse time will require use of a ReverseBlock I think.